src/coap_blockwise.c

```diff
@@ -39,6 +42,15 @@ struct blockwise_transfer
     void *callback_arg;
 };
 
+struct blockwise_transfer_multi
```

Comment:
Since this is the public facing struct, let's keep it simple and clear for users, and rename the context struct for the single shot API.
```suggestion
struct blockwise_transfer
```